events/events-team/projectmanagement/README.md

```diff
@@ -7,6 +7,9 @@ The Project Manager(s) is responsible for Triaging the project board, related is
 Also, keep track of the timelines of these tasks and outlined activited. 
 The Project Manager role will ideally have a shadow for succession planning. Those who have been a part of a previous event get first dibs on lead roles to maintain consistency.
 
+## Shadow to the Project Manager Role
+This role can include a shadow role as well. The PM can delegate/assign the tasks to the shadow.The expectation from a shadow is to lead one of the events-team in the next summit. The shadow is responsible for handling all Project Management related activities in the absence of the PM.
```

Comment:
```suggestion
This role can include one or more shadows. The PM can delegate/assign tasks to the shadows. The expectation from a shadow is to lead one of the events-team roles in an upcoming summit. The shadows are responsible for handling all Project Management related activities in the absence of the PM.
```

Similar to the above, just some suggestions to make it clear that one or more shadows are possible 👍